actioncable/test/channel/stream_test.rb

```diff
@@ -200,7 +200,7 @@ def open_connection
       end
 
       def receive(connection, command:, identifiers:, channel: "ActionCable::StreamTests::ChatChannel")
-        identifier = JSON.generate(channel: channel, **identifiers)
+        identifier = JSON.generate(identifiers.merge(channel: channel))
```

Comment:
This changed merging order, but I confirmed that changing order in this helper method has no effect.